x-pack/plugins/security_solution/server/endpoint/services/artifacts/manifest_manager/manifest_manager.ts

```diff
@@ -500,62 +557,84 @@ export class ManifestManager {
    * @returns {Promise<Error[]>} Any errors encountered.
    */
   public async tryDispatch(manifest: Manifest): Promise<Error[]> {
-    const errors: Error[] = [];
-
+    const allPackagePolicies: PackagePolicy[] = [];
     await iterateAllListItems(
       (page) => this.listEndpointPolicies(page),
-      async (packagePolicy) => {
-        // eslint-disable-next-line @typescript-eslint/naming-convention
-        const { id, revision, updated_at, updated_by, ...newPackagePolicy } = packagePolicy;
-        if (newPackagePolicy.inputs.length > 0 && newPackagePolicy.inputs[0].config !== undefined) {
-          const oldManifest = newPackagePolicy.inputs[0].config.artifact_manifest ?? {
-            value: {},
-          };
-
-          const newManifestVersion = manifest.getSemanticVersion();
-          if (semver.gt(newManifestVersion, oldManifest.value.manifest_version)) {
-            const serializedManifest = manifest.toPackagePolicyManifest(packagePolicy.id);
-
-            if (!manifestDispatchSchema.is(serializedManifest)) {
-              errors.push(
-                new EndpointError(
-                  `Invalid manifest for policy ${packagePolicy.id}`,
-                  serializedManifest
-                )
-              );
-            } else if (!manifestsEqual(serializedManifest, oldManifest.value)) {
-              newPackagePolicy.inputs[0].config.artifact_manifest = { value: serializedManifest };
-
-              try {
-                await this.packagePolicyService.update(
-                  this.savedObjectsClient,
-                  // @ts-expect-error TS2345
-                  undefined,
-                  id,
-                  newPackagePolicy
-                );
-                this.logger.debug(
-                  `Updated package policy ${id} with manifest version ${manifest.getSemanticVersion()}`
-                );
-              } catch (err) {
-                errors.push(err);
-              }
-            } else {
-              this.logger.debug(
-                `No change in manifest content for package policy: ${id}. Staying on old version`
-              );
-            }
+      (packagePolicy) => {
```

Comment:
the page size in `itereateAllListItems` should be updated, too - it still expects 20 items per page

I'm concerned that this is not found by the tests, maybe it would be useful to add a (failing) test for this
and also a `PAGE_SIZE` constant to use everywhere in this file

Reply:
This has been updated in prev commits